### PR TITLE
Add fr.rubet.rpn

### PR DESCRIPTION
**Pull request by louisrubet**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by louisrubet** [comment]
bot, build

**Comment by hfiguiere** [comment]
bot, build fr.rubet.rpn

File: fr.rubet.rpn.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+app-id: fr.rubet.rpn
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: rpn
+modules:
+  - mpfr.yaml
```
Comment (hfiguiere) [inline fr.rubet.rpn.yml:7]: this does not need to be in a separate yml file.

File: fr.rubet.rpn.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - name: rpn
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/louisrubet/rpn.git
+        tag: v2.4
+        commit: bafc48ab85e06003d38b4cba1a95fcbc4562acd5
+    post-install:
```
Comment (hfiguiere) [inline fr.rubet.rpn.yml:15]: there is no appstream fiel nor desktop file.

You never install the icons (only 128x128 or scalable SVG is required)

Comment (louisrubet) [reply fr.rubet.rpn.yml:15]: you are super reactive! :+1: 
on my way

Comment (louisrubet) [reply fr.rubet.rpn.yml:15]: > there is no appstream fiel
What do you mean by `no appstream`? Is my app ID wrong?

> nor desktop file.
This app is a cli, do I need to add a desktop file?

Comment (louisrubet) [reply fr.rubet.rpn.yml:15]: (found it in the doc)

**Comment by louisrubet** [comment]
@hfiguiere @barthalion

My soft systematically receives a SIGKILL after a few minutes, whether it is run by `flatpak run fr.rubet.rpn` or by the desktop link. This doesn't happen when it is run in a non-containerized environment.

Could it be linked to the flatpak environment ?

**Comment by louisrubet** [comment]
bot, build fr.rubet.rpn

File: fr.rubet.rpn.yml (hunk 1 above)
Comment (louisrubet) [reply fr.rubet.rpn.yml]: done

**Comment by louisrubet** [comment]
> @hfiguiere @barthalion
> 
> My soft systematically receives a SIGKILL after a few minutes, whether it is run by `flatpak run fr.rubet.rpn` or by the desktop link. This doesn't happen when it is run in a non-containerized environment.
> 
> Could it be linked to the flatpak environment ?

2 things I found about this issue:
- rpn gets a SIGKILL because all the flatpak environment gets down (bwrap and xdg-dbus-proxy processes go down)
- this happen ONLY in wayland, all is working fine under X11.

I am available to help you move forward on this subject.

**Comment by louisrubet** [comment]
> @hfiguiere @barthalion
> 
> My soft systematically receives a SIGKILL after a few minutes, whether it is run by `flatpak run fr.rubet.rpn` or by the desktop link. This doesn't happen when it is run in a non-containerized environment.
> 
> Could it be linked to the flatpak environment ?

Nevermind, that was a background-run permission problem, as exposed in https://github.com/flatpak/xdg-desktop-portal/issues/478.

**Comment by louisrubet** [comment]
bot, build fr.rubet.rpn

**Comment by louisrubet** [comment]
bot, build fr.rubet.rpn

**Pull request closed without merging**